src/networking/postApiRequest.ts

```diff
@@ -17,9 +17,9 @@ const postApiRequest = async (request: MCMCMonitorRequest): Promise<MCMCMonitorR
         }
     )
     const response = await rr.json()
-    if (!isMCMCMonitorResponse) {
+    if (!isMCMCMonitorResponse(response)) {
```

Comment:
The other bug--as originally written, this was only testing whether the `isMCMCMonitorResponse` function was defined--it wasn't actually applying it to the response itself.